src/startup.rs

```diff
@@ -0,0 +1,47 @@
+use std::path::Path;
+use std::io::Read;
+
+use crate::HEMTTError;
+use crate::error::PrintableError;
+
+macro_rules! exec {
+    ($c:expr) => {
+        std::thread::spawn(|| {
+            $c().unwrap_or_print();
+        })
+    };
+}
+
+pub fn startup() {
+    exec!(check_git_ignore);
+    exec!(deprecated);
+}
+
+/// Checks for the recommended items in a .gitignore
+/// Display a warning if they are not found
+fn check_git_ignore() -> Result<(), HEMTTError> {
+    if Path::new(".gitignore").exists() {
+        let mut data = String::new();
+        open_file!(".gitignore")?.read_to_string(&mut data)?;
+        let mut ignore = vec!("releases/*", "*.biprivatekey");
```

Comment:
Private key is in releases anyways.

Reply:
https://github.com/synixebrett/HEMTT/blob/50eb56845eb3914b9659355a2d8aa6a94e7c497a/src/commands/build/postbuild/sign.rs#L25

The keys should be backed up somewhere anyway, but this will try to prevent people from losing their private key. I was going to inquire about adding `keys/` too

Reply:
Since the only time the private key would be written to disk, is if they want to reuse it.

Reply:
Yeah, makes sense.